### ISSUE TITLE
Performance without BERT fine-tuning

### ISSUE THREAD

**Issue by shinying** [jayleicn/TVQAplus#16]
Hi @jayleicn,

After carefully reading all previous discussions about fine-tuning format, unfortunately I still cannot reproduce your performance with BERT fine-tuned myself. BERT without fine-tuning gives around 67%, but fine-tuning with subtitles and QAs even leads to lower accuracy (66%) then with subtitles only (68%) . Thus I'm wondering if you have conducted experiments without any fine-tuning as well as subtitles-only fine-tuning. Then I can make sure if any problem occurs in my text feature extraction.

Thanks a lot.

**Comment by jayleicn** [comment]
Hi @shinying,

Thanks for your interest in our work. I actually never tried using BERT features without any finetuning or finetuning only on subtitles. It is possible that the model will perform better in one of the scenarios. Besides, did you use all the supervision available, i.e., QA, Grounding, Temporal Localization?  Meanwhile, could you elaborate on your way of extracting features and finetuning the BERT? 

Best,
Jie

**Comment by shinying** [comment]
Hi @jayleicn,

Sorry for the delayed reply. I found that some of my BERT extracted features have different shape with the released ones, and those cases contain words with apostrophes like _n't_ or _'s_ which are split by the tokenizer without ### as _n_, _'_, _t_ or _'_, _s_. However, in my first implementation I only average embeddings of unseen tokens starting with ###. After I average embeddings of tokens tokenized with apostrophes as well, the performance is quite close to the reported number.

Thanks for all your help!

**Comment by jayleicn** [comment]
@shinying Ah, glad to hear that! Let me know if you need any other help.
